crates/optimism/node/src/args.rs

```diff
@@ -66,6 +67,13 @@ pub struct RollupArgs {
     /// Minimum suggested priority fee (tip) in wei, default `1_000_000`
     #[arg(long, default_value_t = 1_000_000)]
     pub min_suggested_priority_fee: u64,
+
+    /// A URL pointing to a secure websocket subscription that streams out flashblocks.
+    ///
+    /// If given, the flashblocks are received to build pending block. All request with "pending"
+    /// block tag will use the pending state based on flashblocks.
+    #[arg(long)]
+    pub flashblocks_url: Option<Url>,
```

Comment:
we have String value for other endpoint args, which would then fail during setup, but also those are just http endpoints,

validating this on cli level seems better actually, so we can keep this url